### DISCUSSION TITLE
MEW links to add Network to wallet

### DISCUSSION THREAD

**Discussion by manumeleone** [community/community#21428] [Other Feature Feedback, Questions, & Ideas]
<p>I need to add network (custom RPC) to my MEW wallet so I can interact with a staking contract for MultiVac crypto.  This can be done easily on Metamask but it seems that option is no longer available with MEW. I have found via MEW help center a link to Github for adding a network. But, link yields page reading “this is not page you are looking for” and in upper left "branch manager has been moved to main "  PleASE HELP</p>

**Comment by manumeleone** [comment]
<p>Nothing that pops up here seems related or I am not familiar with terms. I am not experienced with Github at all.</p>

**Comment by Zerotask** [comment, marked as answer]
<p>This is a general Support Forum for git and GitHub related questions, but not for every single project hosted on GitHub.<br>
Try to use Google or any other search engine to find a more appropriate page</p>
